playbook_snapshot/lib/src/snapshot_device.dart

```diff
@@ -1,30 +1,44 @@
+import 'package:flutter/cupertino.dart';
 import 'package:flutter/rendering.dart';
 
-enum SnapshotDevice {
-  iPhoneSE,
-  iPhone8,
-  iPhone11,
-  pixel4,
-  xperia,
-}
+class SnapshotDevice {
+  const SnapshotDevice({
+    required this.name,
+    required this.size,
+    required this.platform,
+  });
+
+  final String name;
+  final Size size;
+  final TargetPlatform platform;
```

Comment:
Currently `platform` is not used in snapshot test.
Seems we can set `debugDefaultTargetPlatformOverride = device.platform` in here:

https://github.com/playbook-ui/playbook-flutter/blob/8dad6f0893c677863b5e6c0b3f22d305de780e00/playbook_snapshot/lib/src/snapshot.dart#L35

Please check whether it works or not.

Reply:
Fixed it but it shows garbage character when defaultTargetPlatform is iOS.